src/assets/blockly-authoring/toolbox/molasses-toolbox.xml

```diff
@@ -0,0 +1,90 @@
+<xml id="toolbox" style="display: none">
+  <category name="Volcano" colour="#EB0000">
+    <block type="molasses_simulation_all_params">
+      <value name="molasses_eruption_volume">
+        <shadow type="math_number">
+          <field name="NUM">200000000</field>
+        </shadow>
+      </value>
+      <value name="molasses_lava_front">
+        <shadow type="math_number">
+          <field name="NUM">5</field>
+        </shadow>
+      </value>
+      <value name="molasses_vent_location">
+        <shadow type="lat_long">
+          <field name="lat_long">19.5, -155.565</field>
+        </shadow>
+      </value>
+    </block>
+    <block type="molasses_simulation_eruption_volume">
+      <value name="molasses_eruption_volume">
+        <shadow type="math_number">
+          <field name="NUM">200000000</field>
+        </shadow>
+      </value>
+    </block>
+    <block type="molasses_simulation_lava_front">
+      <value name="molasses_lava_front">
+        <shadow type="math_number">
+          <field name="NUM">5</field>
+        </shadow>
+      </value>
+    </block>
+  </category>
+
+  <category name="Logic" colour="%{BKY_LOGIC_HUE}">
+    <block type="controls_if"></block>
+    <block type="logic_compare"></block>
+    <block type="logic_operation"></block>
+    <block type="logic_negate"></block>
+    <block type="logic_boolean"></block>
+  </category>
+
+  <category name="Loops" colour="%{BKY_LOOPS_HUE}">
+    <block type="controls_repeat_ext">
+      <value name="TIMES">
+        <shadow type="math_number">
+          <field name="NUM">10</field>
+        </shadow>
+      </value>
+    </block>
+    <block type="controls_whileUntil"></block>
+    <block type="controls_for">
+      <value name="FROM">
+        <shadow type="math_number">
+          <field name="NUM">1</field>
+        </shadow>
+      </value>
+      <value name="TO">
+        <shadow type="math_number">
+          <field name="NUM">10</field>
+        </shadow>
+      </value>
+      <value name="BY">
+        <shadow type="math_number">
+          <field name="NUM">1</field>
+        </shadow>
+      </value>
+    </block>
+  </category>
+
+  <category name="Data" colour="%{BKY_MATH_HUE}">
+    <block type="text">
+      <field name="TEXT">Name</field>
+    </block>
+    <block type="math_number">
+      <field name="NUM">4</field>
+    </block>
+    <block type="input_range"></block>
+    <block type="lat_long"></block>
+  </category>
+
+  <category name="Variables" colour="%{BKY_VARIABLES_HUE}" custom="VARIABLE"></category>
+
+  <category name="Functions" colour="%{BKY_PROCEDURES_HUE}" custom="PROCEDURE"></category>
+  <!-- <category name="Logging" colour="260">
+    <block type="logprint"></block>
+    <block type="stringconcat"></block>
+  </category> -->
```

Comment:
Do we think this might be coming back or can it be removed?

Reply:
I'm not sure, but it's already in `full-toolbox.xml` so no need to keep it here.